config/locales/views/settings/en.yml

```diff
@@ -39,6 +39,7 @@ en:
             you will have to do so in your settings for the specific provider:
           confirm: Are you absolutely sure you want to remove your %{provider} account?
           submit: Remove %{provider}
+          unable_to_remove: You are unable to remove one of your authentication methods as you need a remaining method to authenticate you. Please add another authentication method above before continuing.
```

Comment:
Thanks for using the yml space to set this value! 

Would you mind adding this `unable_to_remove` key to `account.remove` under `views/settings/fr.yml`?

According to Google translate, a good starter value would be 
```
Vous ne pouvez pas supprimer l'une de vos méthodes d'authentification car vous avez besoin d'une méthode restante pour vous authentifier. Veuillez ajouter une autre méthode d'authentification ci-dessus avant de continuer.
```

(I notice that most of the fr.yml's file's values are currently in English, with a few exceptions. We've been trying to get basic French into the localization as we go along. Hopefully there will be an opportunity for a bigger refactor down the road.)